### PR TITLE
Support for producing WebSpecs using ReSpec

### PR DESCRIPTION
**Pull request by darobin**

### CODE REVIEW

File: js/w3c/headers.js [hunk 1]
```diff
@@ -238,24 +245,42 @@ define(
...
                 msg.pub("start", "w3c/headers");
 
                 // Default include RDFa document metadata
                 if (conf.doRDFa === undefined) conf.doRDFa = true;
                 // validate configuration and derive new configuration values
-                if (!conf.license) conf.license = "w3c";
-                // NOTE: this is currently only available to the HTML WG
-                // this check will be relaxed later
-                conf.isCCBY = conf.license === "cc-by" && conf.wgPatentURI === "http://www.w3.org/2004/01/pp-impl/40318/status";
+                if (!conf.license) conf.license = conf.specStatus === "webspec" ? "w3c-software" : "w3c";
```
Comment (marcoscaceres) [inline js/w3c/headers.js]: Nit: for legibility,  `(conf.specStatus === "webspec")`

File: js/w3c/style.js [hunk 2]
```diff
@@ -31,6 +31,15 @@ define(
                     // with a PNG of the same base name. CONNEG must die.
                     css += "www.w3.org/community/src/css/spec/" + statStyle.toLowerCase() + ".css";
                 }
+                else if (statStyle === "webspec") {
+                    css = "https://specs.webplatform.org/assets/css/kraken.css";
+                    $('<link rel="shortcut icon" href="https://specs.webplatform.org/assets/img/icon.png">', doc)
```
Comment (marcoscaceres) [inline js/w3c/style.js]: I don't think you need "shortcut".

File: js/w3c/templates/webspecs-headers.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<header>
+  <a href="http://specs.webplatform.org/" id="logo">
+    <img alt="Web Platform Specs logo" height="50" src="https://specs.webplatform.org/assets/img/logo.svg" width="50"><br>
```
Comment (marcoscaceres) [inline js/w3c/templates/webspecs-headers.html]: Nit: `<br>` ... and `span` :( The markup for this header seems... um, yucky.

**Comment by marcoscaceres** [comment]
Apart from the nits, it looks great.

File: js/w3c/headers.js (hunk 1 above)
Comment (darobin) [inline js/w3c/headers.js]: Done.

File: js/w3c/style.js (hunk 2 above)
Comment (darobin) [inline js/w3c/style.js]: True, it's still in my coding reflexes from 15 years ago. Need to get those checked.

File: js/w3c/templates/webspecs-headers.html (hunk 3 above)
Comment (darobin) [inline js/w3c/templates/webspecs-headers.html]: Yeah, the `span` was a leftover from other things. The `br` is needed though, this is poetry.

**Pull request merged**